src/views/txHashDecoder.js

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from "react";
+import { Button, Card, CardHeader, CardBody, Row, Col, Label } from "reactstrap";
+import '../../assets/css/black-dashboard-react.css';
+import Caver from 'caver-js'
+import { networkLinks } from '../constants/klaytnNetwork'
+import Column from "../components/Column"
+let caver;
+
+class txHashDecoder extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            network: "mainnet",
+            result: "",
+            rawResult: "",
+            txHash: ""
+        }
+    }
+
+    componentDidMount(){
+        caver = new Caver(new Caver.providers.HttpProvider(networkLinks[this.state.network]["rpc"]))
+    }
+
+    handleInputChange = (e) => {
+        const { name, value } = e.target;
+        this.setState({
+            [name]: value
+        })
+    }
+
+    handleNetworkChange = (e)=>{
+        caver = new Caver(new Caver.providers.HttpProvider(networkLinks[e.target.value]["rpc"]))
+        this.setState({
+            network: e.target.value
+        })
+    }
+
+    decodeTxHash = async() => {
+        const { txHash } = this.state;
+        try{
+            const res = await caver.transaction.getTransactionByHash(txHash);
+            if (res){
+                this.setState({ result: JSON.stringify(res, null, 2), rawResult: res.getRawTransaction() })
+            }else{
+                this.setState({ result: "" })
+            }
+        }
+        catch(err){
+            this.setState({ result: "[ERROR] PLEASE ENTER THE CORRECT VALUE OF TRANSACTION HASH!" })
+        }
+    }
+
+    copyTxToClipboard = (e)=>{
+        const el = this.tx
+        el.select()
+        document.execCommand("copy")
+    }
+
+    copyRawTxToClipboard = (e)=>{
+        const el = this.rawTx
+        el.select()
+        document.execCommand("copy")
+    }
+
+    render() {
+        const { txHash, result, rawResult } = this.state;
+        return (
+            <div>
+                <Column>
+                    <Card>
+                        <CardHeader>
+                            <h3 className="title">TxHash Decoder</h3>
+                            <p style={{color:"#6c757d"}}>
+                            The tool was designed to get a transaction from the transaction hash(TxHash).
```

Comment:
How about changing this sentence to "On this page, you can query a transaction by transaction hash from the Klaytn network to get a caver transaction instance." ? Please let me know your thought.

Reply:
I think, that sentence is more better to understand